cypress-io/cypress
# fix: AUT URL in runner header links to application URL

- Closes #21823

### User facing changelog


Runner header now links to application URL while running e2e tests

### Additional details


Before v10, the application URL in the AUT header was a link that the user could click which would open the current application URL in a new tab. That functionality was lost during updates to the header for v10.

### Steps to test


1. Run an e2e test
2. In the header "address bar", click the application URL
3. Verify that a new tab is opened at the application URL that was open in the runner during your test

### How has the user experience changed?


**Before**
![Screen Shot 2022-06-07 at 3 38 06 PM](https://user-images.githubusercontent.com/14275198/172468209-9248e334-48cd-43a4-a802-8200004e99d0.png)

**After**
![Screen Shot 2022-06-07 at 3 32 00 PM](https://user-images.githubusercontent.com/14275198/172468298-01622822-ae5b-4fc0-bf57-f8eebda7d8f5.png)


### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
